### DISCUSSION TITLE
wtf syncing speed

### DISCUSSION THREAD

**Discussion by nguyenhathienphu** [Chia-Network/chia-blockchain#8775] [Support]
my computer lost power suddenly, after 2h i turned it on. and it took 30min to connect to the sever, and took 5 min to sync the data of only 2h. 
how can i get that sync speed faster ????????????????????????

**Comment by BrandtH22** [comment]
@nguyenhathienphu , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
